### ISSUE TITLE
Difficulty using flags

### ISSUE THREAD

**Issue by GilPalma** [HybridEidolon/psobbaddonplugin#7]
Hey! I've been working with this repo and Soly's to play around with addons for PSO. I've only been improving on stuff already been made but I've made some substantial superficial upgrades.

I've been having some problems with flags, however, when trying to add menu bars to windows and, most recently, working with text inputs. I can't seem to understand how to use them or even if they've been fully implemented yet. In the file 'master/bbmod/src/wrap_imgui_impl.cpp' there seem to be wrappers of some kind that adapt the C++ flags to Lua. However, I don't understand how to use them, especially since there's a comment saying '// Enums not handled by iterator yet' which I don't understand.

tl;dr: How do I apply flags to function calls?

P.S.: If you need help with your rework, I'm somewhat available to help. Not sure I'm qualified or have enough time but I'll try to pitch in if you need help.

**Comment by HybridEidolon** [comment]
In the C++ API of ImGui, flags are combined with bitwise operators. In the Lua API, use their names as strings in an array table, ala:

```lua
local flags = {'NoTitleBar', 'AlwaysAutoResize'}
imgui.Begin('a window', true, flags)
```

Sorry this took a while to respond to.
